argilla/docs/how_to_guides/import_export.md

```diff
@@ -120,27 +120,64 @@ dataset = rg.Dataset.from_hub(repo_id="<my_org>/<my_dataset>")
 
 The `rg.Dataset.from_hub` method loads the configuration and records from the dataset repo. If you only want to load records, you can pass a `datasets.Dataset` object to the `rg.Dataset.log` method. This enables you to configure your own dataset and reuse existing Hub datasets. See the [guide on records](record.md) for more information.
 
+
 !!! note "With or without records"
 
-    The example above will pull the dataset's `Settings` and records from the hub. If you only want to pull the dataset's configuration, you can set the `with_records` parameter to `False`. This is useful if you're just interested in a specific dataset template or you want to make changes in the dataset settings and/or records.
+    === "Argilla configured dataset"
 
-    ```python
-    dataset = rg.Dataset.from_hub(repo_id="<my_org>/<my_dataset>", with_records=False)
-    ```
+        The example above will pull the dataset's `Settings` and records from the hub. If you only want to pull the dataset's configuration, you can set the `with_records` parameter to `False`. This is useful if you're just interested in a specific dataset template or you want to make changes in the dataset settings and/or records.
+
+        ```python
+        dataset = rg.Dataset.from_hub(repo_id="<my_org>/<my_dataset>", with_records=False)
+        ```
 
-    With the dataset's configuration, you could then make changes to the dataset. For example, you could adapt the dataset's settings for a different task:
+        With the dataset's configuration, you could then make changes to the dataset. For example, you could adapt the dataset's settings for a different task:
 
-    ```python
-    dataset.settings.questions = [rg.TextQuestion(name="answer")]
-    dataset.update()
-    ```
+        ```python
+        dataset.settings.questions = [rg.TextQuestion(name="answer")]
+        dataset.update()
+        ```
 
-    You could then log the dataset's records using the `load_dataset` method of the `datasets` package and pass the dataset to the `rg.Dataset.log` method.
+        You could then log the dataset's records using the `load_dataset` method of the `datasets` package and pass the dataset to the `rg.Dataset.log` method.
 
-    ```python
-    hf_dataset = load_dataset("<my_org>/<my_dataset>")
-    dataset.records.log(hf_dataset)
-    ```
+        ```python
+        hf_dataset = load_dataset("<my_org>/<my_dataset>")
+        dataset.records.log(hf_dataset)
```

Comment:
Should we mention adding a mapping here in case column names are different? e.g. I use a dataset as template but take the data from elsewhere.